src/_includes/common/getting-started.md

```diff
@@ -104,16 +110,17 @@ Performance monitoring helps developers measure Apdex, Throughput, and trace slo
 
 To get started with performance monitoring, first install the `@sentry/tracing` package:
```

Comment:
```suggestion
To get started with performance monitoring, first install the `@sentry/apm` package:
```

Reply:
Thanks. FYI - In the final phase, we will align this with the Performance page content.